docs/pages/docs/guide/advanced/table.mdx

```diff
@@ -173,16 +173,6 @@ Just wrap your table with curly braces `{` and `}`, e.g.
 </table>}
 ```
 
-Alternatively, create a reusable table component:
-
-```mdx filename="MDX"
-export function Table({ columns, data }) {
-  return <table>{/* ... */}</table>
-}
-
-<Table columns={...} data={...} />
-```
-
```

Comment:
@87xie fyi this was removed in MDX 3.1 🥴 https://github.com/mdx-js/mdx/issues/2549